tests/steps/database.py

```diff
@@ -111,13 +112,16 @@ def get_config_option(self, option):
         return deepcopy(self.cursor.fetchone())['opt']
 
     def create_replication_slot(self, slot_name):
-        self.cursor.execute(
-            """
-            SELECT pg_create_physical_replication_slot(%(name)s)
-        """,
-            {'name': slot_name},
-        )
-        return deepcopy(self.cursor.fetchone())
+        try:
+            self.cursor.execute(
+                """
+                SELECT pg_create_physical_replication_slot(%(name)s)
+            """,
+                {'name': slot_name},
+            )
+            return deepcopy(self.cursor.fetchone())
+        except DuplicateObject:
+            return True
```

Comment:
Maybe return `[True]` to mimc structure of one row returned by `.fetchone()`?

Reply:
ok, but it's does not matter.
there is only one call of this function
https://github.com/yandex/pgconsul/blob/main/tests/steps/cluster.py#L331